### PR TITLE
Add documentation for onboarding a provider with metadata service

### PR DESCRIPTION
**Pull request by abhat**
This commit enhances the onboarding guide with necessary information
to secure a infrastructure provider's metadata service.

Signed-off-by: Aniket Bhat <anbhat@redhat.com>

### CODE REVIEW

**Comment by abhat** [comment]
@elmiko @candita. Reopened the PR against the official repo.

**Comment by elmiko** [comment]
thanks @abhat 

prior discussion here https://github.com/elmiko/ocp-infrastructure-provider-onboarding-guide/pull/32

File: docs/infrastructure-metadata-service/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+metadata service IP based on your platform to the [generateIPTablesRules](https://github.com/openshift/sdn/blob/master/pkg/cmd/openshift-sdn-cni/openshift-sdn.go#L121) function.
+
+### OVN Kubernetes
+
+To block access to the metadata service from pods in OpenShift, please add your
+metadata service IP based on your platform to the [setupIPTablesBlocks](https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/cni/OCP_HACKS.go#L35) function.
+
+### Questions
+
+Questions can be directed to the OpenShift Slack channel **#forum-sdn**
```
Comment (bostrt) [inline docs/infrastructure-metadata-service/index.md:26]: Is this channel accessible by users outside Red Hat?

Comment (abhat) [reply docs/infrastructure-metadata-service/index.md:26]: No, I assume we would have guest/partner slack accounts for infra provider engineers working with OCP Engineering?

Comment (bostrt) [reply docs/infrastructure-metadata-service/index.md:26]: In the past, I've seen us setup dedicated Slack channels that partner's can join as a "single channel guest". But I'm not aware of partner's being able to join channels like #forum-sdn.

Comment (elmiko) [reply docs/infrastructure-metadata-service/index.md:26]: we might just want to mention reaching out to their red hat partner for questions. it seems some like of the relationships result in partner slack channels, and others do not.

**Review by elmiko — COMMENTED** [review]
i'm generally good with this patch, once we address @bostrt 's concern, i'm good to merge.

**Comment by abhat** [comment]
Pushed the change to address @bostrt 's concern. @elmiko please merge if this looks good.

**Review by elmiko — APPROVED** [review]
thanks @abhat 
/approve

**Comment by abhat** [comment]
@elmiko do we need to set up merge bot on this repo?

**Comment by elmiko** [comment]
@abhat i _thought_ it was setup. guess i missed something

**Comment by elmiko** [comment]
this should do the trick, https://github.com/openshift/release/pull/27032

**Pull request merged**